configure-oidc.md

```diff
@@ -75,3 +75,191 @@ When issuing a `cosign sign` command, add the flags to pass in the client secret
+| `Client secret`    | the client secret from your GitHub OAuth app's page (note: you may need to generate a new secret to obtain this value) |
+| `Enabled`          | `ON`                                                                                                                   |
+| `Trust Email`      | `ON`                                                                                                                   |
+| `First Login Flow` | `first broker login`                                                                                                   |
+
+#### Add IdentityProviderMapper
+
+From the Keycloak console, navigate to `Identity Providers` in the lefthand menu bar, and select the provider you have just created: `github`. Select the tab `Mappers` and click the `create` button on the right-hand side to bring up the following page:
+
+<p align="center">
+  <img src="/images/add_mapper.png" />
+</p>
+
+Set the field values as follows:
+
+| Field                  | Value                                                 |
+| -----------------------|-------------------------------------------------------|
+| `Name`                 | this value can be anything, as it will be overwritten |
+| `Sync Mode Override`   | `inherit`                                             |
+| `Mapper Type`          | `Hardcoded Attribute`                                 |
+| `User Attribute`       | `emailVerified`                                       |
+| `User Attribute Value` | `true`                                                |
+
+
+It should look like this: 
+
+<p align="center">
+  <img src="/images/mapper2.png" />
+</p>
+
+Make sure to hit `save`.
+
+### Copy the Redirect URI to your GitHub OAuth App
+
+From the Keycloak console, navigate to `Identity Providers` in the lefthand menu bar, and select the provider you have just created: `github`. From this page, copy the Redirect URI:
+
+<p align="center">
+  <img src="/images/redirect_uri.png" />
+</p>
+
+Paste the redirect URI value into the OAuth app Authorization Callback URL field. Then paste it into the Homepage URL field, removing `broker/github/endpoint` from the end of the URL.
+
+The values should look like this, where `CLUSTERVALUE` is the unique value from the URL of your OpenShift cluster:
+
+|Field                       | Value|
+|----------------------------|------|
+| Homepage URL               | https://keycloak-keycloak-system.apps.rosa.CLUSTERVALUE.p3.openshiftapps.com/auth/realms/sigstore/ |
+| Authorization Callback URL | https://keycloak-keycloak-system.apps.rosa.CLUSTERVALUE.p3.openshiftapps.com/auth/realms/sigstore/broker/github/endpoint |
+
+
+Add these to the appropriate fields marked below:
+
+<p align="center">
+  <img src="/images/add_urls.png" />
+</p>
+
+Make sure to click `Update Application`.
+
+### Sign the artifact
```

Comment:
Before signing the artifact, the `sigstore` client must be available within the `sigstore` realm. If you are creating the `sigstore` realm from scratch, it will not have the `sigstore` client unless you previously installed RHTAS which included both the realm and client and then created the `sigstore` realm via the CLI. it would then get recreated by the operator.

Regardless, there does need to be some wording that describes steps that the user needs to take regardless of the approach

Reply:
This is a good point, I'm putting a note in about the assumptions re: installation and will ping to see what you think.